java2typescript-jackson/src/test/resources/java2typescript/jackson/module/WriterPreferencesTest.enumToStringLiteralType.d.ts

```diff
@@ -3,5 +3,11 @@ export type Enum =
     | "VAL2"
     | "name";
 
+export interface TestClassHasMapWithEnumKey {
+    enumKeyMap: { [key in Enum ]: string;};
```

Comment:
Good catch! This is actually a relatively simple change (if I understand your concern): `enumKeyMap: { [key in Enum] ? : string; }` with the added `?` will make it so that not all enum values are required. I can make that change as well.